community/samples/serving/helloworld-deno/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM hayd/alpine-deno:0.38.0
+EXPOSE 8080
+WORKDIR /app
+# Prefer not to run as root.
+USER deno
```

Comment:
While not running as root is a good practice, we mostly skip this for simplicity unless the container contents need it.

Reply:
Done.